packages/wallet-sdk/src/sign/walletlink/relay/connection/HeartbeatWorker.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2018-2025 Coinbase, Inc. <https://www.coinbase.com/>
+
+
+/**
+ * This worker is used to send heartbeat messages to the main thread.
+ * It is used to keep the websocket connection alive when the webpage is backgrounded.
+ * 
+ */
+
+const HEARTBEAT_INTERVAL = 10000; // 10 seconds
+
+type WorkerMessage = {
+  type: 'start' | 'stop';
+}
+
+export type WorkerResponse = {
+  type: 'heartbeat'
+}
+
+let heartbeatInterval: NodeJS.Timeout | undefined;
+
+// Listen for messages from the main thread
+self.addEventListener('message', (event: MessageEvent<WorkerMessage>) => {
```

Comment:
I don't know how long WL is going to stay around, so in the mean time can we add some basic tests around this?

Reply:
Added!